### PR TITLE
Add actions and update dependencies

### PR DESCRIPTION
**Pull request by NivedhaSenthil**

### CODE REVIEW

File: .github/workflow/tests.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13.1
+
+      - name: Setup java
+        uses: actions/setup-java@v1
+        with:
+          java-version: '12'
+
+      - uses: bugdiver/setup-gauge@master
```
Comment (zabil) [inline .github/workflow/tests.yml:33]: Should be move this repo to getgauge?

**Review by zabil — CHANGES_REQUESTED** [review]

File: .github/workflow/tests.yml (hunk 1 above)
Comment (NivedhaSenthil) [reply .github/workflow/tests.yml:33]: cc @BugDiver

Comment (BugDiver) [reply .github/workflow/tests.yml:33]: Sure We can do that.

Comment (BugDiver) [reply .github/workflow/tests.yml:33]: Have transferred the repo. Please update the links

**Pull request closed without merging**